.changeset/gold-tigers-rhyme.md

```diff
@@ -0,0 +1,5 @@
+---
+'@rocket.chat/meteor': patch
+---
+
+fix: dynamic slug during search
```

Comment:
please rephrase the changeset  to match the pr title

Reply:
@ggazzo changesets updated